packages/core/helper-plugin/package.json

```diff
@@ -69,8 +69,8 @@
     "@storybook/builder-webpack5": "6.5.9",
     "@storybook/manager-webpack5": "6.4.10",
     "@storybook/react": "^6.5.10",
-    "@strapi/design-system": "1.2.7",
-    "@strapi/icons": "1.2.7",
+    "@strapi/design-system": "1.3.0-beta.3",
+    "@strapi/icons": "1.3.0-beta.3",
```

Comment:
Just to make sure because @joshuaellis has approved it already: do we want to have a beta release in the `main` branch or should we wait for a final `1.3.0` release?

Reply:
I believe the plan is (or should be) when it's been QA'd & ready to merge from all aspects, to release 1.3.0 stable and include it here before merging to main. @simotae14 is gonna talk to me when he's ready for that, if i've understood everything correctly.

Reply:
Ok. I'll add a no merge label instead. Thanks!

Reply:
yes, now I have updated with the version 1.4.0